superset/reports/dao.py

```diff
@@ -138,7 +139,6 @@ def create(cls, properties: Dict[str, Any], commit: bool = True) -> Model:
         create a report schedule and nested recipients
         :raises: DAOCreateFailedError
         """
-        import json
```

Comment:
I think cases like these are motivation enough for enabling this check - I assume this was unintentional, and was probably a misclick in an IDE (this context menu from PyCharm when clicking Option+Return): 
![image](https://user-images.githubusercontent.com/33317356/129528629-d505abaa-87ad-4ea4-9acc-ff05fa819d4f.png)

Reply:
@villebro agreed, though as pointed out in the description this check highlights the cyclical dependencies which unnecessarily exist and we should strive to eradicate.